src/Escaping/SynapseQuote.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TableBackendUtils\Escaping;
+
+class SynapseQuote implements QuoteInterface
+{
+    public static function quote(string $value): string
+    {
+        return "'" . str_replace("'", "''", $value) . "'";
+    }
+
+    public static function quoteSingleIdentifier(string $str): string
+    {
+        return '[' . str_replace(']', '][', $str) . ']';
```

Comment:
Přišlo mi to divný, tak jsem se podíval jak to dělá SQLServer a dělá to jinak... 

`SELECT QUOTENAME('abc[]def');`

`[abc[]]def]`

A tohle by udělalo `[abc[][def]`, ne @zajca 

Ale jestli je to intentional, tak OK...

Reply:
https://docs.microsoft.com/en-us/sql/t-sql/functions/quotename-transact-sql?view=sql-server-ver15

Reply:
Zajímavé, podle toho to dělá špatně doctrine z tama sem to vykopčil https://github.com/doctrine/dbal/blob/3.1.x/src/Platforms/SQLServer2012Platform.php#L1585
Zkůšal sem to a máš pravdu, naštěstí sa asi ještě nenašel masochysta co by měl `[]` v názvu tabulky

Reply:
Vypálil sem bug report https://github.com/doctrine/dbal/issues/4643